src/Utilities/FlowAnalysis/Analysis/PointsToAnalysis/PointsToAnalysis.PointsToDataFlowOperationVisitor.cs

```diff
@@ -640,7 +640,12 @@ private PointsToAbstractValue GetValueBasedOnInstanceOrReferenceValue(IOperation
                 switch (nullState)
                 {
                     case NullAbstractValue.NotNull:
-                        return defaultValue.MakeNonNull(operation, DataFlowAnalysisContext);
+                        if (!AnalysisEntityFactory.TryCreate(operation, out var analysisEntityOpt))
+                        {
```

Comment:
Won't analysisEntityOpt be null if TryCreate returns false?

Reply:
Yes, let me convert this to an assert.